app/utils/query-converter.js

```diff
@@ -77,9 +86,8 @@ const METRIC_TEST = new RegExp(`${METRIC}(?:,*${METRIC})*`, 'i');
 export default class QueryConverter {
   // BQL to Query methods
 
-  static classifyBQL(select, groupBy) {
-    // Group By
-    if (!isEmpty(groupBy)) {
+  static classifyBQL(select, groupBy, innerSelect, extraGroupBy) {
+    if (!isEmpty(extraGroupBy)) {
```

Comment:
extra is a good word for the things with a `2` 🤷

Reply:
going with "extra" in front of the regex group names. X instead of 2 for the regex const names.. I guess?